packages/core/src/base/index.scss

```diff
@@ -1,3 +1,3 @@
 @import 'body';
-@import 'typography';
+@import 'typography/typography';
```

Comment:
Moved typography to its own folder due to splitting up styles and examples into their own files